ROADMAP.md

```diff
@@ -0,0 +1,43 @@
+# Roadmap
+
+# Phase 1 (v0.1.0):
+
+- [ ] Create a basic Bevy project and import the necessary dependencies.
+- [ ] Create a simple 3D scene with a camera and light.
+- [ ] Import a 3D model from a third-party software.
+- [ ] Display the 3D model in the scene.
+
+# Phase 2 (v0.2.0):
+
+- [ ] Implement a system for adding joints to imported models.
+- [ ] Implement a system for controlling joints with keyboard keys.
+- [ ] Add support for different types of joints (revolute, fixed, spherical, etc.).
+- [ ] Add support for attaching keyboard keys to motors.
+
+# Phase 3 (v0.3.0):
+
+- [ ] Implement more advanced features for controlling joints, such as speed and precision.
+- [ ] Add support for multiple joints in a single model.
+- [ ] Add support for nested joints.
+- [ ] Control spawn camera position and projection.
+
+# Phase 4 (v0.4.0):
+
+- [ ] Add support for more complex interactions between joints.
+- [ ] Add support for physics simulation of joints.
```

Comment:
I am a bit confused by this. Isn't Bevy already taking care of the physics and isn't the joint physics partially working in the sample you showed me?

Reply:
You're completely right, I forgot to delete this step in this phase.